andela/ah-frontend-zeus
# #162163134 create and view articles:

**What does this PR do? Allow Authors to create articles**

- Allow Authors to create an article
- Allow Authors to view all articles
- Allow Authors to view a single article

 **Description of Task to be completed?**
Our application has no UI for creating and viewing articles. This task allows us to create a UI to create an article, view a single article and view all articles already created.

**How should this be manually tested?**

- Install dependencies: `yarn`
- Run the application server: `yarn start`
- To create an article go to `article/post` in the browser
- To see all articles go to `/articles` in the browser
- To see a single article click on that article to display it
- To edit an article click on the edit icon

**What are the relevant pivotal tracker stories?**
[ #162163134](https://www.pivotaltracker.com/story/show/162163134)

**Screenshots**
create an article:
<img width="1435" alt="screen shot 2019-01-17 at 9 29 56 am" src="https://user-images.githubusercontent.com/9946845/51306735-25c64480-1a4f-11e9-8e97-5009af1392eb.png">
view article feed:
<img width="1437" alt="screen shot 2019-01-17 at 9 29 06 am" src="https://user-images.githubusercontent.com/9946845/51306768-34146080-1a4f-11e9-81ee-c140b89250e1.png">
view a single article:
<img width="1436" alt="screen shot 2019-01-17 at 9 59 29 am" src="https://user-images.githubusercontent.com/9946845/51306816-47bfc700-1a4f-11e9-906b-f3bfc6196ddc.png">
